app/src/main/java/io/horizontalsystems/bankwallet/modules/ratelist/RateListFragment.kt

```diff
@@ -42,9 +42,12 @@ class RatesListFragment : Fragment() {
     }
 
     private fun observeView(view: RateListView?) {
-        view?.datesLiveData?.observe(viewLifecycleOwner, Observer { (date, lastUpdateTimestamp)->
-            dateText.text = DateHelper.formatDate(date, "MMM dd")
-            setLastUpdatedTime(lastUpdateTimestamp)
+        view?.datesLiveData?.observe(viewLifecycleOwner, Observer { lastUpdateTimestamp->
+            if (lastUpdateTimestamp == null)
```

Comment:
May be using optional let is better here?

Reply:
Even better would be the none-optional argument in here: ```observeView(view: RateListView)```